CHANGELOG.md

```diff
@@ -9,7 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Removed
 
+- Stop logging debug messages when metrics are collected.
```

Comment:
I'd call it fixed. Because it is not a breaking change. Rather an improvement.

My proposition:

```suggestion
### Fixed

- Reduce noisy debug logging when metrics are collected.
```